src/TabBar.js

```diff
@@ -70,7 +71,7 @@ export default class TabBar<T: Route<*>>
     renderLabel: PropTypes.func,
     renderIndicator: PropTypes.func,
     onTabPress: PropTypes.func,
-    tabStyle: View.propTypes.style,
+    tabStyle: ViewPropTypes.style,
```

Comment:
This is only present in RN 0.44, this breaks the library for all older versions.